data/capbase.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Capbase",
+  "url": "https://capbase.com/",
+  "career_page_url": "",
+  "type": "Product",
+  "categories": ["startup"],
```

Comment:
Also allowed categories are the following -> 

> Allowed Company Categories
> - cloud_software
> - design_ux
> - marketing_writing
> - hr

Reply:
Sorry for the mistake! Fixed